documentation/tools/dataprotector/methods/transferownership.md

```diff
@@ -0,0 +1,46 @@
+# transferOwnership
+
+Allows transferring ownership of protected data to a new owner, identified by their ETH address. The return value provides a transaction hash and confirmation of the new owner of the data.
+
+## Usage
+
+```javascript
+const transferResponse = await dataProtector.transferOwnership({
+    protectedData: '0xA0Cf798816D4b9b9866b5330EEa46a18382f251e',
+    newOwner: '0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e',
+});
+```
+
+## Return value example
+
+```javascript
+[
+    {
+        "address": "0xA0Cf798816D4b9b9866b5330EEa46a18382f251e",
```

Comment:
When calling the method: `protectedData`
When getting the response, naming becomes `address`?

Maybe keep the property name as `protectedData` also in the response?